### ISSUE TITLE
Selection outline jitters when TAA is enabled

### ISSUE THREAD

**Issue by jdswebb** [turanszkij/WickedEngine#116]
Per title.

**Comment by turanszkij** [comment]
Yes, I know about this. The selection outline is rendered after TAA, but uses stencil buffer. To solve this, there are two options:
- move before TAA pass. In this case it would have to be part of the engine, instead it is only an editor feature right now. And then also tested for every render path
- or apply a second TAA pass for outline and keep it editor only feature
Tbh I didn't want to do either, but it's something to consider.

**Comment by turanszkij** [comment]
It's fine as it is for an editor-only feature, closing.
